analysis_options.yml

```diff
@@ -1 +1 @@
-include: package:pedantic/analysis_options.yaml
+include: package:pedantic/analysis_options.1.9.0.yaml
```

Comment:
Should this version be `1.10.0` for null safety?

Reply:
File `analysis_options.1.10.0.yaml` does not exists, https://github.com/google/pedantic/tree/master/lib